source/includes/fact-system-vars.rst

```diff
@@ -0,0 +1,108 @@
+.. list-table::
+   :widths: 30 10 40 10 10
+   :stub-columns: 1
+   :header-rows: 1
+
+   * - System Variable
+     - Data Type
+     - Description
+     - Default Value (Atlas-hosted)
+     - Default Value (On Premises)
+   * - ``full_pushdown_exec_mode``
+     - boolean
+     - Specifies whether a query error is returned for queries with
+       clauses that aren't fully pushed down. SQL query predicates
+       usually appear in ``WHERE`` clauses and are used to filter data
+       returned by the query.
+     - ``0``
+     - ``0``
+   * - ``log_level``
+     - integer
+     - Specifies the logging level for |bi-short|:
+
+       - ``-1``: Do not log.
+       - ``0``: Log only messages for database users, such as basic
+         ``mongosqld`` events and state changes.
+       - ``1``: Log messages for database users and ``mongosqld`` administrators.
+       - ``2``: For internal use only.
+     - ``2``
+     - ``0``
+   * - ``max_nested_table_depth``
+     - integer
+     - Specifies the maximum number of unique nested field paths that
+       ``mongosqld`` maps to a relational table for a collection.
+     - ``50``
+     - ``50``
+   * - ``max_num_columns_per_table``
+     - integer
+     - The maximum number of unique fields that ``mongosqld`` maps to
+       relational columns for a collection.
+     - ``1000``
+     - ``2000``
+   * - ``mongodb_max_varchar_length``
+     - integer
+     - Specifies the maximum string length returned for columns using
+       the ``VARCHAR`` data type. A value of ``0`` specifies
+       no limit.
+     - ``0``
+     - ``0``
+   * - ``polymorphic_type_conversion_mode``
+     - string
+     - Determines how MongoDB evaluates document fields that are
+       specified with multiple data types. For example, ``count``
+       could be a number in one document and a string in another.
+
+       - ``off``:
+         Queries may fail if you do not explicitly cast
+         document fields that are specified with multiple data types.
+       - ``fast``:
+         |bi-short| casts document fields as appropriate
+         if MongoDB samples at least two documents during schema
+         generation that use different data types for the same field.
+         In the example, MongoDB would cast ``count`` as an integer.
+       - ``safe``:
+         |bi-short| automatically casts all document fields
+         that are specified with multiple data types.
+     -  ``off``
+     -  ``off``
+   * - ``sample_refresh_interval_secs``
+     - integer
+     - Specifies how frequently, in seconds, that the |bi-short| schema
+       is updated. A value of ``0`` specifies that the schema is not 
+       refreshed after |bi-short| starts.
+
+       See :doc:`Cached Sampling </schema/cached-sampling>` for more
+       information.
+     - ``0``
+     - ``300``
```

Comment:
I believe this has recently changed - see https://github.com/10gen/mms/commit/9fc3179326f2997f8f9c8f9a6a918fd774b9de11

You might want to confirm with the Atlas team though.

Reply:
Ralph C. confirms. Fixed.